fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatPermissionEvaluator.java

```diff
@@ -148,6 +150,22 @@ private static RetryHandler buildRetryHandler(
   @Override
   public boolean hasPermission(
       Authentication authentication, Object resource, Object authorization) {
+    if (!fiatStatus.isEnabled()) {
```

Comment:
Does it make sense to use the `services.fiat.granted-authorities.enabled` flag here too?

Reply:
Hmm, maybe? Though when the flag is disabled, this method would end up returning false anyways since the authentication won't have any granted authorities. I can update this.